scripts/create_tables.sql

```diff
@@ -1 +1,30 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY
```

Comment:
This and the subsequent attempts at creating tables won't execute since there's a missing comma after `KEY`.